config/webpack/environment.js

```diff
@@ -1,3 +1,10 @@
 const { environment } = require('@rails/webpacker')
-
+const webpack = require('webpack')
+environment.plugins.prepend('Provide',
+    new webpack.ProvidePlugin({
+        $: 'jquery',
+        jQuery: 'jquery',
+        Popper: ['popper.js', 'default']
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/rikai-academy/booking_tours_01/pullRequest?prid=7834702)
```suggestion
        Popper: ["popper.js", 'default']
```